### ISSUE TITLE
A suspicious bug. 

### ISSUE THREAD

**Issue by zqudm** [SpoonLabs/gumtree-spoon-ast-diff#247]
String c3= " class X { public  X(){ this(duration, null, null)}}";
                 String c4= " class X { public X(){ super();}}";
   when compairing the string c3 and c4.  It outputs  the following, 

Delete FieldRead at X:1
	duration
Delete Literal at X:1
	null
Delete Literal at X:1
	null
//////////////////////////////////////////////////

this seems buggy?

**Comment by algomaster99** [comment]
Hey @zqudm ! I will revert to this by tomorrow for sure. However, till then, could you please correct the syntax of code in `c3`? I see `duration` is not defined anywhere and `;` is missing after `null)`. Our tool does not guarantee correct AST diff for the syntactically incorrect programs.

**Comment by algomaster99** [comment]
I inspected this again with the following test case:
```java
String c3= " class X { public  X(){ this(); }}";
String c4= " class X { public X(){ super(); }}";
```
It shows no AST change so it is indeed a bug.

**Comment by algomaster99** [comment]
@zqudm You issue was fixed in #248. Please let me know if it was fixed for your experiment. You can download `v1.52` by registering the following specification in your pom.

```xml
<dependency>
  <groupId>fr.inria.gforge.spoon.labs</groupId>
  <artifactId>gumtree-spoon-ast-diff</artifactId>
  <version>1.52</version>
</dependency>
```
